dist/education-notice/education-notice.css

```diff
@@ -27,6 +28,9 @@
   -webkit-margin-start: var(--spacing-150);
           margin-inline-start: var(--spacing-150);
 }
+.education-notice button.icon-btn {
+  background-color: transparent;
```

Comment:
Can we use `icon-btn--transparent` modifier instead? 
https://github.com/eBay/skin/blob/master/src/less/icon-button/icon-button.less#L63

We are doing this in other places, like infotip or date-textbox.

Reply:
Ah, yes, will do. Good catch!

Reply:
It looks like the transparent modifier brings in a negative outline offset change (no idea why it's there) that screws things up pretty bad. Seems like a possible rabbit hole and out of scope. Maybe we should just go out with what we have for now?
<img width="761" alt="image" src="https://github.com/eBay/skin/assets/1675667/bb782848-c1e1-438c-b8c6-9f4e3c2183ab">

Reply:
It looks like `infotip` and likely others are overwriting over the offset to make things look normal. 
<img width="305" alt="image" src="https://github.com/eBay/skin/assets/1675667/4d322643-4cca-4490-be3c-50ffe2319cc9">

Reply:
Yeah we should figure that out. 
Lets go with this and add an issue to fix icon button and all those other ones. I've seen lots of those issues with icon button before.